src/client/datascience/notebook/helpers/helpers.ts

```diff
@@ -361,6 +361,14 @@ const orderOfMimeTypes = [
     'application/json',
     'text/plain'
 ];
+function isEmptyVendoredMimeType(outputItem: NotebookCellOutputItem) {
+    if (outputItem.mime.startsWith('application/vnd.')) {
```

Comment:
Do they always start with ```application/vnd```? Would ```application``` also apply?

Reply:
Yes, they always start with `application/vnd` for vendor specific mime types.